pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint_test.go

```diff
@@ -165,7 +164,11 @@ func TestGetRemoteEntrypoint(t *testing.T) {
 	}))
 	defer server.Close()
 	image := path.Join(strings.TrimPrefix(server.URL, "http://"), "image:latest")
-	ep, err := entrypoint.GetRemoteEntrypoint(entrypoint.NewCache(), image)
+	entrypointCache, err := NewCache()
+	if err != nil {
+		t.Errorf("couldn't new entrypoint cache: %v", err)
```

Comment:
i think the word "create" (or similar) missing from this sentence and a couple other similar errors, e.g.:
"couldn't create new entrypoint cache:"